working-open.md

```diff
@@ -11,6 +11,7 @@ Hypha operates with democratic organizing principles informed by our [values](./
 # How and what?
 
 - We currently accommodate redactions and going off-the-record for meeting notes. Participants can request to stop taking notes, or redact and correct notes after they have been taken.
+- We recognize that some information should be treated as sensitive and confidential for the benefit of Hypha and its clients and partners. An employee or contractor with access to such information is responsible for keeping this information confidential.
```

Comment:
What is a "partner" here? think we're following out of sync with previous preferred term "collaborator", we should update either this page, or this one: https://handbook.hypha.coop/co-operative.html#about-us

Reply:
Sticking with what's existing.

```suggestion
- We recognize that some information should be treated as sensitive and confidential for the benefit of Hypha and its clients and collaborators. An employee or contractor with access to such information is responsible for keeping this information confidential.
```

Reply:
Nevermind that, see below.